release/package.json

```diff
@@ -6,7 +6,9 @@
   },
   "activationEvents": [
     "onCommand:workbench.action.tasks.runTask",
+    "onDebugInitialConfigurations:coreclr",
     "onDebugDynamicConfigurations:coreclr",
+    "onDebugResolve:coreclr",
```

Comment:
These are nice-to-haves - making sure that we are initialized whenever coreclr launch types are used